src/test/java/htsjdk/samtools/BAMRemoteFileTest.java

```diff
@@ -42,7 +42,7 @@
 public class BAMRemoteFileTest {
     private final File BAM_INDEX_FILE = new File("src/test/resources/htsjdk/samtools/BAMFileIndexTest/index_test.bam.bai");
     private final File BAM_FILE = new File("src/test/resources/htsjdk/samtools/BAMFileIndexTest/index_test.bam");
-    private final String BAM_URL_STRING = "http://www.broadinstitute.org/~picard/testdata/index_test.bam";
+    private final String BAM_URL_STRING = "https://personal.broadinstitute.org/picard/testdata/index_test.bam";
```

Comment:
Can `https://personal.broadinstitute.org/picard/testdata/` be declared as a shared constant somewhere so that we don't have to do this again?